JQ.sublime-syntax

```diff
@@ -24,14 +24,18 @@ contexts:
   ## Blocks
   enter:
     - match: '\('
+      scope: punctuation.section.parens.begin.jq
```

Comment:
Something about the updates seems to break the parenthesis matching for interpolated strings

On tidy branch:
![image](https://github.com/zogwarg/SublimeJQ/assets/6682272/5bd472e8-5d35-435b-bb95-7b2543a96739)

On master branch:
![image](https://github.com/zogwarg/SublimeJQ/assets/6682272/3b93a1f0-53d7-4b7c-8e6a-149e431c86fc)

It would be nice to fix before merging and release, also as todos (which I can do):
- Add new builtin and keywords where applicable
- Update coloring.png to be less quirky and reflect new highlighting.

Reply:
interesting, I can't reproduce that on Sublime Text build 4155.
![Screenshot from 2023-09-19 23-21-28](https://github.com/zogwarg/SublimeJQ/assets/11882719/e66f5038-145d-41bd-a649-76b5d83230f0)